.run-clang-format

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+find . -type f -regex '.*\.\(cpp\|hpp\|cc\|cxx\|h\|hxx\)' -exec clang-format-8 -style=file -i {} \;
```

Comment:
What are "cc" files? or did you mean to type "c"?

Reply:
For the GNU GCC compiler the file extension cc is a recognized for c++